.github/workflows/ci.yml

```diff
@@ -27,10 +27,13 @@ jobs:
     name: Build and Test
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
A matter of personal style:

I have been hosed a number of times by using `foo-latest` rather than, say `ubuntu-20.04`.   

`foo-latest` is good for rapid prototyping but fragile going beyond that .

GitHub can and does change the `-latest`.  Things can go
bump in the night where the shrapnel leaves little direct clue.

Yes, this introduces the technical debt of having to manually
change all the `ubuntu-18.04` to `ubuntu-20.04` and to 
figure out what the desired `-mumble` is.

Reply:
Yes, absolutely agree. Let's get these pinned down!